nailgun/entities.py

```diff
@@ -1892,6 +1893,22 @@ def clone(self, synchronous=True, **kwargs):
         response = client.post(self.path('clone'), **kwargs)
         return _handle_response(response, self._server_config, synchronous)
 
+    def generate(self, synchronous=True, **kwargs):
```

Comment:
Add coverage for this method https://github.com/SatelliteQE/nailgun/blob/master/tests/test_entities.py#L338